context/app/static/js/theme/theme.tsx

```diff
@@ -248,6 +252,19 @@ const theme = createTheme({
         underline: 'none',
       },
     },
+    MuiButton: {
+      variants: [
+        {
+          props: { variant: 'elevated' },
+          style: {
+            boxShadow:
+              '0px 1px 3px 0px rgba(0, 0, 0, 0.20), 0px 2px 2px 0px rgba(0, 0, 0, 0.12), 0px 0px 2px 0px rgba(0, 0, 0, 0.14)',
+            borderRadius: '8px',
+            fontWeight: 300,
```

Comment:
Does it make sense to include the `borderRadius` and `fontWeight` in the new variant? These seem to be specific to each use?

Reply:
I moved these out into a styled component 👍🏻